java/pom.xml

```diff
@@ -144,11 +144,23 @@
         <skipNativeCopy>false</skipNativeCopy>
         <cxx11.abi.value>ON</cxx11.abi.value>
         <cxx.flags/>
+        <CMAKE_EXPORT_COMPILE_COMMANDS>OFF</CMAKE_EXPORT_COMPILE_COMMANDS>
         <CUDA_STATIC_RUNTIME>OFF</CUDA_STATIC_RUNTIME>
         <PER_THREAD_DEFAULT_STREAM>OFF</PER_THREAD_DEFAULT_STREAM>
+        <project.build.directory>target</project.build.directory>
```

Comment:
Why are we setting the project build directory at all for the default profile?

Reply:
@revans2 I have removed the change to the project build directory. PTAL